TypeApprovalRegister.yaml

```diff
@@ -38,21 +38,21 @@ paths:
                   type: string
```

Comment:
The changes for /v1/bequeath-your-data-and-die looks fine for all attributes except for new-application-name.
Requesting you to consider https://github.com/openBackhaul/ApplicationPattern/pull/469#discussion_r1052346862 before closing this changes.

Reply:
The attribute holding "NewRelease" must be updated by the actual application name.
If you would see differently, please let me know in a separate email

Reply:
Currently, the application-name of *-1-0-0-http-c-nr-1-0-0-000 is defined as "NewRelease". Changing the new-application-release to "update" looks fine and does not seem to have any problem because transactions happens through forwardings of /v1/bequeath-your-data-and-die. But, is this an expected behavior to update NewRelease value in http-client and not maintaining it. 
Kindly let me know your views.

Reply:
Yes, it is the expected behavior that content of object with UUID _\*-1-0-0-http-c-nr-1-0-0-000_ in CONFIG will be updated by content of RequestBody of _/v1/bequeath-your-data-and-die_.

Example:
The HttpClient with UUID _tar-2-0-0-http-c-nr-2-0-0-000_ contains an _application-name_ attribute that currently holds the value "NewRelease", if this value would not be updated by content of RequestBody of _/v1/bequeath-your-data-and-die_ the old release would create subscriptions for an application named "NewRelease" at other applications during executing the bequeath-your-data-and-die process.

Reply:
Many thanks for your precise reviewing.